web/public/index.html

```diff
@@ -28,6 +28,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>the survey</title>
+    <script src="https://unpkg.com/pace-js@1.2.4/pace.min.js"></script>
```

Comment:
For every ajax request, it doesn't have to show all restarted progress bar on every ajax update. So disable with `data-pace-options='{ "restartOnRequestAfter": false }'` property on this script. See https://github.com/CodeByZach/pace#configuration.

And also Youtube doesn't show every restarted progress bar on every ajax udpate.